### ISSUE TITLE
application delete doesn't work

### ISSUE THREAD

**Issue by ich777** [PushBits/cli#6]
Hi,
I've encounter a strange issue currently, when I try to do:
`pbcli application delete --url https://<redacted> --username <username> 15`

I get a error 500 back:
`FATA Request failed with HTTP 500 Internal Server Error. `

Also noticed that it says FATA instead of FATA**L** too. Am I doing something wrong? I deleted a few applications back then and it was working fine.
In the above example `15` is the id from the application itself, it also doesn't matter if I do it like that:
`pbcli application delete 15 --url https://<redacted> --username <username>`

EDIT: I forgot to say that I can create new applications just fine.

Regards,
Christoph

**Comment by ich777** [comment]
@eikendev just a little update, I've now deleted the application in the database itself, this worked but I would rather prefer if the issue would be fixed, or maybe I missed something at the CLI.
Please let me know if you need anything to debug this issue.

**Comment by eikendev** [comment]
Hi @ich777, sorry for the late reply. I cannot reproduce this issue, I can delete an application just fine with pbcli. What database and OS do you use?

**Comment by ich777** [comment]
No worries @eikendev 
I use the default database and Pushbits is running on Unraid (Slackware Linux). As stated above I have now delete this one app in the database itself, please also note that I've not done anything in the database prior...

Back when I installed Pushbits and made the template available to users for Unraid deleting was working fine. If you want me to I can create a new application and try to delete it again. I can also send you the database structure if needed.

**Comment by eikendev** [comment]
Hi @ich777, please try run a fresh PushBits instance and create a new application there. Please also provide the version of PushBits and pbcli you use. Does the debug log show anything besides the HTTP 500?

I just checked on my PushBits and a fresh install, and don't get this error. Without being able to replicate the issue, I'll close for now but re-open once we know it's an error of PushBits or pbcli themself.

**Comment by ich777** [comment]
The main issue that I have with that is that I can't easily do that because I have many services (about 15 I think) where I can't easily change the key because I use PushBits heavily - thank you BTW for that wonderful software, was searching a long time for a good alternative to Pushover and Gotify.

I just use the latest version from your Docker container, I've also tried many other different versions in terms of `pbcli` and also on different OS (Windows/Linux).

Maybe my database is corrupt but I have to say that deleting it through a SQL browser was working flawlessly.

Is it maybe possible that this is caused because I've set up 5 services back in the day and then deleted the 2nd one and now I have basically a "hole" in between application 1 and 3 where a service is missing <- I hope I explained that good enough.

Again, creating applications is just working fine.
